### ISSUE TITLE
ifcopenshell.version prints 0.0.0 in FreeCAD

### ISSUE THREAD

**Issue by semhustej** [IfcOpenShell/IfcOpenShell#5811]
I reported issue to FreeCAD GitHub and it was tagged by developers as an IfcOpenShell bug: https://github.com/FreeCAD/FreeCAD/issues/18141

Steps to reproduce:

Type in Python console:
- import ifcopenshell
- print(ifcopenshell.version)
- 0.0.0 is returned which is not the correct version number

Apparently the IfcOpenShell version is 0.8.0 (using FreeCAD 1.0)

**Comment by Krande** [comment]
@semhustej Which distribution of ifcopenshell are you using? The conda-forge distribution or the compiled versions of ifcopenshell-python from github? If I recall correctly, freecad might be relying on the conda-forge package?

I ask because the conda-forge feedstock does not (yet) add the version to the package init. The ifcopenshell-python compiled on github on the other hand include the version in the makefile -> https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.8.0/src/ifcopenshell-python/Makefile#L118 .

It's been a while since the v0.8.0 ifcopenshell release. So hopefully we'll see a v0.8.1 tagged soon. As part of v0.8.1 release I can include the version specifier in the conda-forge distribution as well.

**Comment by semhustej** [comment]
Hi @Krande ,

thanks for looking at the issue. I am afraid I am not knowledgable enough to answer your question with autority. I am using official release of FreeCAD 1.0.

From the weekly FreeCAD builds it seems like it is built with conda:
![weekly-builds](https://github.com/user-attachments/assets/fcf9dee9-eb84-4bd7-ba8b-1f88435c4c1d)


It would be certainly helpful for FreeCAD users to be able to display the version number. The version number (0.0.0) is also exported to the IFC files created by IfcOpenShell through FreeCAD, which is a bit confusing.

**Comment by Krande** [comment]
@semhustej I have updated the conda-forge package of ifcopenshell now with the included version specifier.
The fix is included from version v0.8.0*_6 (ie . py313h1400811_6) and will be included in all future conda-forge versions.

```
(ifctest) C:\Work\code>python
Python 3.13.1 | packaged by conda-forge | (main, Dec  5 2024, 21:02:16) [MSC v.1942 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import ifcopenshell
>>> print(ifcopenshell.version)
0.8.0
```

**Comment by semhustej** [comment]
Thank you, @Krande !
